lib/matplotlib/text.py

```diff
@@ -312,6 +315,23 @@ def get_rotation_mode(self):
         "get text rotation mode"
         return self._rotation_mode
 
+    def set_trans_rotate(self, tr):
+        """
+        set the boolean value which represents whether the text
+        is to be rotated with a given transformation matrix, or
+        not
+
+        """
+        if tr is None or tr in [True, False]:
```

Comment:
I think else where we are very forgiving about 'truthy' values, probably should be here as well (and just cast through `bool`).